### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by alvinometric**
Updated the readme to mention how to grab variants, most of our other SDKs mention it there

### CODE REVIEW

**Review by gastonfournier — APPROVED** [review]
LGTM! 
cc @melindafekete just for awareness but I think the PR makes sense

File: README.md [hunk 1]
```diff
@@ -146,6 +146,20 @@ the event that all your app does is to start the SDK and check a feature status,
 node app running until the SDK has synchronized with the Unleash API. It is **not** required in
 normal apps.
 
+#### Check variants
+
+You can use the `getVariant` method to get the variant of a feature flag. If the flag is disabled or
```
Comment (melindafekete) [inline README.md:151]:
```suggestion
You can use the `getVariant` method to retrieve the variant of a feature flag. If the flag is disabled or
```

File: README.md [hunk 2]
```diff
@@ -146,6 +146,20 @@ the event that all your app does is to start the SDK and check a feature status,
...
+if it has no variants, then you will get back the
```
Comment (melindafekete) [inline README.md:152]:
```suggestion
doesn't have any variants, the method returns the
```

File: README.md [hunk 3]
```diff
@@ -146,6 +146,20 @@ the event that all your app does is to start the SDK and check a feature status,
...
+[disabled variant](https://docs.getunleash.io/reference/feature-toggle-variants#the-disabled-variant)
```
Comment (melindafekete) [inline README.md:153]:
```suggestion
[disabled variant](https://docs.getunleash.io/reference/feature-toggle-variants#the-disabled-variant).
```

**Review by melindafekete — DISMISSED** [review]

**Pull request merged**